client/src/MovementDispatcher.cpp

```diff
@@ -0,0 +1,70 @@
+//
+// Created by tdancois on 1/14/17.
+//
+
+#include <EntityManager.h>
+#include <iostream>
+#include "MovementDispatcher.h"
+
+MovementDispatcher::MovementDispatcher(const int32_t refreshRateMs)
+    : refreshRateMs(refreshRateMs),
+      needUpdate(false) {}
+
+void MovementDispatcher::updateNetwork(const PlayerCommands &playerCommands,
+                                       LockedQueue<AresProtocol::AresMessage> &sendingQueue) {
+  sf::Time deltaTime = clock.restart();
```

Comment:
If the loop is always faster thant the refreshRate, the movements are never broadcast to the network.

Use [sf::Clock::getElapsedTime](https://www.sfml-dev.org/documentation/2.0-fr/classsf_1_1Clock.php#a799feb6acb099b57b58d8d20984fce11) instead.